sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md

```diff
@@ -35,6 +30,9 @@
   Code has been updated to [1.0.0-beta.8](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/Resources.Azure-1.0.0-beta.8/src/OpenTelemetry.Resources.Azure).
   ([#44682](https://github.com/Azure/azure-sdk-for-net/pull/44682))
 
+* Removed an experimental feature for logs emitted within an active tracing context to follow the Activity's sampling decision.
+  ([#44745](https://github.com/Azure/azure-sdk-for-net/pull/44745))
```

Comment:
nit: Consider including this message:
> (This feature was originally introduced in 1.2.0-beta.1)

I say this because we added it a while ago, removed it for the stable release, added it back, and are now removing it again. 😅 Adding the extra context may help users understand what this feature was.

Reply:
sure